src/components/ReservedRockets.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import './styles/reserved.css';
+
+const ReservedRockets = () => {
+  const { rockets } = useSelector((state) => state.rockets);
+  const reservedRockets = rockets.filter((list) => list.reserved === true);
+
+  return (
+    <sections>
+      <h2>My Rockets</h2>
+      <ul className="mission-list">
+        {reservedRockets.map((list) => {
+          const { id, name } = list;
+          return <li key={id} className="mission-name">{name}</li>;
+        })}
+      </ul>
+    </sections>
+  );
+};
+
```

Comment:
- [ ] Rocket reservation is not working as intended. When rockets are reserved and then a user visits the profile page and back, the reservation check is cleared from the rocket page as well as the profile page.

[rocket.webm](https://github.com/jecode93/react-group-project/assets/12745474/bf0dd3e2-114e-4d67-8f88-d7ae8628342b)